2015/presentations/Business logic with PostgreSQL and Python/text.md

```diff
@@ -0,0 +1,144 @@
+# Business logic with PostgreSQL and Python
+
+## Introduction
+
+In my opinion PostgreSQL is the most advanced open source database. It has so many powerful features that is really difficult to put all of them here in this one acticle.
+
+Here below I'm going to show you the least known feature of PostgreSQL which is procedural languages and stored functions. By using those stored procedures we are going to build business logic step by step.
+
+## Business logic
+
+By this term we are going to call all of those blocks of code which are responsible for making calculations and taking decisions about data flow. In many cases most of developers decide to build business logic as a part ORM models. It is pretty convenient. Whenever data is being used from database ORM can take control and decide how data is going to be returned to upper objects. Some ORM frameworks like Django allow us to extend base model and start building our own logic in a very complex way.

```

Comment:
This term we are going to use to refer to all code of block responsible for making calculations